osu.Game/Overlays/BeatmapSet/Buttons/HeaderDownloadButton.cs

```diff
@@ -27,7 +29,7 @@ public class HeaderDownloadButton : BeatmapDownloadTrackingComposite, IHasToolti
 
         private readonly bool noVideo;
 
-        public LocalisableString TooltipText => button.Enabled.Value ? "download this beatmap" : "login to download";
+        public LocalisableString TooltipText => BeatmapsetsStrings.ShowDetailsDownloadDefault;
```

Comment:
I am guessing the login text was trimmed here because the entire overlay goes blank if the user is logged out?